veracruz-test/src/main.rs

```diff
@@ -111,7 +111,12 @@ mod tests {
             env_logger::builder().init();
             let _main_loop_handle = std::thread::spawn(|| {
                 let mut sys = System::new("Veracruz Proxy Attestation Server");
-                let server = proxy_attestation_server::server::server(actual_server_url, "../test-collateral/CACert.pem", false).unwrap();
+                let server = proxy_attestation_server::server::server(
+                    actual_server_url,
+                    "../test-collateral/CACert.pem",
```

Comment:
Can you factor these out as top-level constants?

Reply:
Can do